src/web/components/MostViewedRightIsland.tsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import { css } from 'emotion';
+
+// Styling the data island root so it stetches to cover the full height available in the container.
+// Requires us to subtract the height of its sibbling in the container (StickyAd).
+const wrapperStyles = css`
```

Comment:
Just a small one. It seems that this naming convention allows us to use the same name everywhere, but it might be better to have more specific variable names

Reply:
Updated class name to reflect purpose.